packages/loot-core/src/server/reports/app.ts

```diff
@@ -15,7 +15,7 @@ import { ReportsHandlers } from './types/handlers';
 
 const reportModel = {
   validate(report: CustomReportEntity, { update }: { update?: boolean } = {}) {
-    requiredFields('reports', report, ['conditions'], update);
+    requiredFields('reports', report, ['conditionsOp'], update);
```

Comment:
cc @carkom the `conditions` field doesn't exist on the `CustomReportEntity` type. LMK if you're ok with the patch I did here